package.json

```diff
@@ -90,12 +90,12 @@
     "ember-data": "^4.11.3",
```

Comment:
We might need to bump ember-data as well. I'm seeing some failures in the embroider tests due to ember-data, maybe a newer 5.x version works properly?